homeyess/website/views.py

```diff
@@ -283,12 +299,18 @@ def delete_ride(request, ride_id):
 def cancel_ride(request, ride_id):
     instance = Ride.objects.get(id=ride_id)
     if instance:
+        if instance.volunteer != None:
+            send_message("Ride to {} has been cancelled".format(instance.interview_address), instance.volunteer)
+        if instance.homeless != None:
+            send_message("Ride to {} has been cancelled".format(instance.interview_address), instance.homeless)
```

Comment:
Can you specify that the volunteer has cancelled the ride and that they will be matched with a different volunteer